mllib/src/main/scala/org/apache/spark/mllib/clustering/StreamingKMeans.scala

```diff
@@ -70,22 +71,38 @@ class StreamingKMeansModel(
     override val clusterCenters: Array[Vector],
     val clusterWeights: Array[Double]) extends KMeansModel(clusterCenters) with Logging {
 
-  /** Perform a k-means update on a batch of data. */
+  /**
+   * Perform a k-means update on a batch of data.
+   * @since 1.2.0
+   */
   def update(data: RDD[Vector], decayFactor: Double, timeUnit: String): StreamingKMeansModel = {
 
-    // find nearest cluster to each point
+    /**
+     * find nearest cluster to each point
+     * @since 1.2.0
+     */
     val closest = data.map(point => (this.predict(point), (point, 1L)))
```

Comment:
Those are local variables or methods inside `update`. It is not necessary to use JavaDoc and add since tags.